src/oc/functabl.cpp

```diff
@@ -120,16 +116,22 @@ void hoc_spec_table(void** vppt, int n) {
         }
         return;
     }
-    if (hoc_is_object_arg(1)) {
+    if (hoc_is_object_arg(1) && hoc_is_object_arg(2)) {
```

Comment:
This addition has been made so in python we can reach the else.
The first argument should be a 1-D array of double, in python this is most of the type an object. But the second value is always a double because this is the size